test/controllers/api/user_images_controller_test.rb

```diff
@@ -4,13 +4,8 @@
   let(:user_image) { create(:user_image) }
 
   it 'should show' do
-    get(:show, { api_version: 'v1', format: 'json', id: user_image.id } )
+    get(:show, { api_version: 'v1', format: 'json', user_id: user_image.user_id } )
```

Comment:
Line is too long. [83/80]<br>Redundant curly braces around a hash parameter.<br>Space inside parentheses detected.